test/dragonfly_libvips/processors/thumb_test.rb

```diff
@@ -9,8 +9,9 @@
   let(:cmyk) { Dragonfly::Content.new(app, SAMPLES_DIR.join('sample_cmyk.jpg')) }
   let(:gif) { Dragonfly::Content.new(app, SAMPLES_DIR.join('sample.gif')) }
   let(:anim_gif) { Dragonfly::Content.new(app, SAMPLES_DIR.join('sample_anim.gif')) }
-  let(:crop_tester) { Dragonfly::Content.new(app, SAMPLES_DIR.join('sample_colors.png')) } # 512x512
+  let(:square_crop_tester) { Dragonfly::Content.new(app, SAMPLES_DIR.join('sample_colors_square.png')) } # 512x512
   let(:portrait_crop_tester) { Dragonfly::Content.new(app, SAMPLES_DIR.join('sample_colors_portrait.png')) } # 512x512
+  let(:landscape_crop_tester) { Dragonfly::Content.new(app, SAMPLES_DIR.join('sample_colors_landscape.png')) } # 512x512
```

Comment:
I've added two more sample images. Most of the tests previously passed for a square image only.